src/main/java/gg/archipelago/client/parts/Game.java

```diff
@@ -18,4 +18,20 @@ public class Game implements Serializable {
 
     @SerializedName("location_name_to_id")
     public HashMap<String,Long> locationNameToId = new HashMap<>();
+
+    @Override
+    public Game clone() {
```

Comment:
why is this being added if clone() is not used anywhere?

Reply:
I forgot to remove it.

Reply:
Should be fixed now.